### PR TITLE
Made modifications to pages

### LINKED ISSUE #158: write up how to reference a local script
Can someone explain to me how to use `InitialWorkDir` requirements without any input parameters.
For instance, I need to put my shell scripts near with `cwl script` and run them from that `cwl script`. Is there some way don't use absolute path to scripts? I tried to use EnvVariable like that:
```
baseCommand: sh
arguments: [$SCRIPT_FOLDER/script.sh]
```
 But it is not working.

Then I found to use InitialWorkDir requirement. I wrote that via that I can put my scripts inside the cwl script and use relative path to shell scripts, but i didn't find any examples how use that without input_parameters.

Anybody ideas?

### PR DESCRIPTION
**Pull request by Mackenzie-OO7**
In this PR, I:
- fixed #158 
- fixed typos and punctuation errors
- modified sentences for better understanding
- embedded link to installing WSL in the **Prerequisites** page

### CODE REVIEW

File: src/introduction/prerequisites.md [hunk 1]
```diff
@@ -91,12 +91,12 @@ You can run the CWL tool description by omitting the `--validate` option:
...
 ### cwl-runner Python module
 
-`cwl-runner` is an implementation-agnostic alias for CWL Runners.
+`cwl-runner` is an implementation-agnostic alias for CWL Runners. This simply means that the `cwl-runner` alias command can be invoked independently, and is not reliant on a particular CWL runner.
 Users can invoke `cwl-runner` instead of invoking a CWL runner like `cwltool`
 directly. The `cwl-runner` alias command then chooses the correct CWL runner.
 This is convenient for environments with multiple CWL runners.
 
-The CWL community publishes a Python package with the name `cwlref-runner` that installs
+The CWL community published a Python package with the name `cwlref-runner` that installs
```
Comment (kinow) [inline src/introduction/prerequisites.md:99]: I think here it should be publishes as it's continuously published?

File: src/faq.md [hunk 2]
```diff
@@ -79,6 +79,55 @@ outputs:
       outputEval: ${self[0].basename=inputs.otu_table_name; return self;}
 ```
 
+## Referencing a local script
+
+We are going to describe two methods of doing this:
+
+The first method involves [adding the folder containing your scripts to the `PATH` environment variable](https://stackoverflow.com/q/8779951/20123733). This allows you to run the shell script without using `sh` or `bash` commands
+
```
Comment (kinow) [inline src/faq.md:87]: Missing period in sentence above, and unnecessary two blank lines (I think... I am using GH mobile but I think a single blank line is OK)

File: src/faq.md [hunk 3]
```diff
@@ -79,6 +79,55 @@ outputs:
...
+
+Start with adding a _shebang_ at the top of your file:
+```
+#!/bin/bash
+```
+After that, make the script executable with command `chmod +x script.sh`
```
Comment (kinow) [inline src/faq.md:93]: /with command/with the command

File: src/faq.md [hunk 4]
```diff
@@ -79,6 +79,55 @@ outputs:
...
+
+We are going to describe two methods of doing this:
+
+The first method involves [adding the folder containing your scripts to the `PATH` environment variable](https://stackoverflow.com/q/8779951/20123733). This allows you to run the shell script without using `sh` or `bash` commands
+
+
+Start with adding a _shebang_ at the top of your file:
+```
+#!/bin/bash
+```
```
Comment (kinow) [inline src/faq.md:92]: Most other examples have blank lines between the code block.

And for code block, I think we should use the `code-block` directive (see other examples for bash... maybe contributing has some info too?) for consistency.

File: src/faq.md [hunk 5]
```diff
@@ -79,6 +79,55 @@ outputs:
...
+After that, make the script executable with command `chmod +x script.sh`
+
+Finally, modify your path to add the directory where your script is located (It is good practice to use `$HOME/bin` for storing your own scripts).
```
Comment (kinow) [inline src/faq.md:95]: s/path/PATH environment variable ?

File: src/faq.md [hunk 6]
```diff
@@ -79,6 +79,55 @@ outputs:
...
+```
+export PATH=$PATH:/appropriate/directory
+```
+
+
```
Comment (kinow) [inline src/faq.md:100]: Extra lines.

File: src/faq.md [hunk 7]
```diff
@@ -79,6 +79,55 @@ outputs:
...
+Now you can use `baseCommand: scriptname.sh.` to run the script directly. When you wish to share your work later, you can place your script in a software container in the Docker format.
+
+
```
Comment (kinow) [inline src/faq.md:103]: Extra lines.

File: src/faq.md [hunk 8]
```diff
@@ -79,6 +79,55 @@ outputs:
...
+outputs: []
+```
+```{note}
+In CWL, everything must be directly stated. 
+Therefore, the script needs to be one of the following:
+- available on the system `PATH` (for cwltool performing local execution, no —preserve-environment is needed)
+- part of the Docker format software container (either a fixed path or once more, on the system `PATH` defined in the container)
+- an input to the CommandLineTool description itself (as shown above).
+```
```
Comment (kinow) [inline src/faq.md:128]: I think these comments are redundant after the text above was added?

File: src/faq.md [hunk 9]
```diff
@@ -79,6 +79,55 @@ outputs:
...
+For further clarification, check out this [issue](https://github.com/common-workflow-language/user_guide/issues/158) in the project repository.
+
```
Comment (kinow) [inline src/faq.md:130]: Project here is not User Guide.. so probably better to say cwltool?

**Review by kinow — COMMENTED** [review]
Looking great! On mobile but tried to leave a quick review. Thanks!!!!

**Comment by Mackenzie-OO7** [comment]
@kinow Thanks for the review, I've made the recommended changes.

File: src/faq.md (hunk 5 above)
Comment (Mackenzie-OO7) [reply src/faq.md:96]: Could you please elaborate? I'm not sure of what exactly you mean here.

**Review by swzCuroverse — COMMENTED** [review]
Looks good.  I just need to make sure it looks OK with read the docs.

**Comment by swzCuroverse** [comment]
I made some small suggestions- once reviewed, happy to merge it.

**Comment by Mackenzie-OO7** [comment]
> I made some small suggestions- once reviewed, happy to merge it.

Hi, I've made the recommended changes.

**Comment by swzCuroverse** [comment]
@tetron This looks good can you merge?

**Review by tetron — APPROVED** [review]

**Pull request merged**